config/300-pullsubscription.yaml

```diff
@@ -28,7 +28,6 @@ spec:
     - all
     - knative
     - pubsub
-    - eventing
```

Comment:
Why remove this?

Reply:
the pullsubscription is not part of eventing, it is part of pubsub was the reasoning. The channel will be part of eventing, and moved to events.cloud.run group.